eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1704,7 +1704,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
   }
 
   def handleCommandError(cause: Throwable, cmd: Command) = {
-    log.error(s"${cause.getMessage} while processing cmd=${cmd.getClass.getSimpleName} in state=$stateName")
+    log.warning(s"${cause.getMessage} while processing cmd=${cmd.getClass.getSimpleName} in state=$stateName")
```

Comment:
How about passing the error to the logger? It will print the stacktrace in the logs and makes it easier to debug errors.
```suggestion
    log.warning(s"${cause.getMessage} while processing cmd=${cmd.getClass.getSimpleName} in state=$stateName", cause)
```

Reply:
We print the stack trace for unknown errors (two lines below). For known errors printing the stack trace doesn't give information and will only pollute the logs.